src/themes/default/modules/accordion.overrides

```diff
@@ -22,5 +22,5 @@
 .ui.accordion .ui.header > .dropdown.icon::before,
 .ui.accordion .title > .dropdown.icon::before,
 .ui.accordion .accordion .title > .dropdown.icon::before {
-    content: '\f0da'/*rtl:'\f0d9'*/;
```

Comment:
Dont remove the rtl hint

Reply:
Sorry about this again, I was expecting a spae added before the comment, but not complete removal. Checking what tool and why it was removed.

Reply:
the comments were removed wrongly by Stylelint `quotes` rule - https://github.com/fomantic/Fomantic-UI/commit/267ddf4af1b63279e11ecac7e6a2ec60788a5682

fixed in https://github.com/fomantic/Fomantic-UI/pull/2612/commits/d68de62a02103d2dc3f752e2f8d83048b232da9b - are the added spaces ok?